src/components/organisms/Navbar/Navbar.js

```diff
@@ -71,45 +71,49 @@ export default function Navbar() {
             <div>
                 {showModal ? (
                 <>
-                    <div className="fixed inset-0 z-50 flex items-center justify-center overflow-y-auto overflow-x-hidden outline-none focus:outline-none">
+                    <div className="fixed inset-0 z-50 flex items-center justify-center overflow-y-auto overflow-x-hidden outline-none focus:outline-none" onClick={() => setShowModal(false)}>
                     <div className="relative mx-auto h-[500px] w-[300px] md:w-[500px]">
                         {/*content*/}
-                        <div className="relative flex w-full flex-col rounded-xl border-0 bg-white shadow-lg outline-none focus:outline-none">
+                        <div className="relative flex w-full flex-col rounded-2xl border-0 bg-white shadow-lg outline-none focus:outline-none">
                         {/*header*/}
-                        <div className="flex w-full rounded-t border-b border-solid border-slate-200">
+                        
+                        {/* Background img */}
+                        <div className="flex w-full rounded-t">
                             <ImageAtomLocal
                             imagesize="pr100"
                             border="none"
                             src={DonateModalImg}
                             alt="Logo Virufy"
                             ImageStyleProps=""
                             />
-                            <div className="absolute flex w-full flex-col items-end p-6">
-                            <button
-                                className="flex h-[35px] w-[35px] justify-center rounded-full bg-white px-1 py-2 font-bold text-black shadow-xl outline-none transition-all duration-150 ease-linear hover:bg-gray-300"
-                                type="button"
-                                onClick={() => setShowModal(false)}
-                            >
-                                X
-                            </button>
+                            {/* Close button */}
+                            <div className="absolute flex w-full flex-col items-end pt-3 pr-5">
+                                <button
+                                    className="flex justify-center rounded-full font-medium text-white text-xl shadow-xl outline-none transition-all duration-150 ease-linear"
```

Comment:
what does `transition-all duration-150 ease-linear` do to the button? I don't notice the effect

Reply:
oh it looks the only change that was made was the formatting.